recipes/pytentiostat/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pytentiostat" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/juami/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 952e6e2743d88a3c519a9644f5514902f4b1726f67221727c71b7a37dd59ea5f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build: []
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - xonsh
+    - pyfirmata
+    - pandas
+    - matplotlib
+    - pyyaml
+
+test:
+  requires:
+    - pytest
+  imports: []
```

Comment:
So if you want to test importing the package, you should add those imports here--and for a pure python package like this, that's probably sufficient. All we really want to do is make sure the package isn't broken. Otherwise, probably best to just remove the section if not doing imports.

Also, right now, all you have said is that `pytest` needs to be installed to run the tests, but you're not actually running any tests. If you really want to run the tests, you'll need to add a `commands` section (or add a `run_test.py`) with a line to run the test suite. Again, IMO this is probably overkill.

Reply:
Thanks @dopplershift this is helpful.  All the tests must pass on all the platforms on GH before the code is released so it is probably overkill here, as you say.  I will remove the test section.  I have removed the empty lists.  I have also updated the requirements that I found were out of date.  I think this is ready for review again.